activities/staff-information/sickness.md

```diff
@@ -45,3 +45,19 @@ This enables us to know you are back in and avoids action by the arbo-dienst.
 
 The operations coordinator immediately reports the sick leave in [the portal of our absenteeism insurance](https://mijnwerkgeversportaal.acumen.nl/index3.html).
 As soon as the employee is better it needs to be reported in the same portal. This can also be done retroactively if the operations coordinator is not present on the day the employee goes back to work.
+
+### Long-term sick leave management
+
+In case an employee is sick for a longer time the ARBO doctor will be involved. Our ARBO service is [RegioPoortwachters](https://www.regiopoortwachters.nl/). A case manager from RegioPoortwachters will reach out to inquire about the sick employee within approx. 10 days after the initial sickness report in the insurance portal. The ARBO service portal is integrated with our insurance portal.
```

Comment:
What is an ARBO doctor?

Reply:
An Arbodienst doctor is a third party organizational doctor. _Arbo_ comes from the Dutch 'Arbeidsomstandighedenwet', shortened to 'arbowet' or employment situation law that tries to safeguard the employment situation of employees.

Reply:
I suggest that this should be included (or linked to) in some way.